tests/fixtures.js

```diff
@@ -21,7 +21,7 @@ define(["a", "b"], function (a, b, c) {
 exports.AMD_DEFINE_WITH_FOO_PLUGIN_AND_JS_EXT = `
 define([
     'foo!aaa/bbb/ccc.js'
-], function () {
+], function (c) {
```

Comment:
Without this change tests were failing with `Should have 1 error but had 0: []`. Most likely related to some checks within [rjs.js](https://github.com/cvisco/eslint-plugin-requirejs/blob/master/lib/utils/rjs.js)

Reply:
I've narrowed the issue down to the `arity` variable which I copied from the other rule implementation. `arity` is the length of arguments from the callback function which was `0` before and `1` after my fixture change. In the next line there was a `slice` which was using the `arity` value and therefore the value of `path` was an empty array.

```js
getDependencyStringNodes(node).slice(0, arity);
```

Maybe best to just review the related fixup commit 4209bc3